### ISSUE TITLE
[invoices] Display unit price on invoices and not the price per quantity ordered

### ISSUE THREAD

**Issue by RachL** [openfoodfoundation/openfoodnetwork#11393]
This one can be done outside the feature toggle: it's something we have missing when working on unit price

## Description

**- As a:** enterprise user or shopper
**- On page:** alternative and regular invoice PDF
**- I want to be able to do:**

Currently we show a price per unit ordered. Example here: 

I have a Melon that is sold in item of 1000 bags (very realistic I know!):

![image.png](https://images.zenhubusercontent.com/595e669a61b4ca3b0a737135/469c40ae-16ac-4604-94e1-977d4f49a6a8)

The shopfront indicates the unit price as being 0,11 cts a bag, which is correct. That's what legally the unit price is about: understanding if the packaging of a product is cheaper than another.

However on the alternative invoice we have a unit price that is based on the quantity ordered (apologies for the screenshot in French, I was displaying staging FR in English though...):

![image.png](https://images.zenhubusercontent.com/595e669a61b4ca3b0a737135/eb13f1c3-4d1f-4408-922c-ec3cf83cfe01)

105 comes from the fact that 1 unit of 1000 bags cost 105. And I ordered only one unit. 105 / 1 = 105. However this is not the legal unit price we must display: we must display the same than in the shopfront.

Let's fix that with this issue. We must display the unit price **excluding tax**

:danger: this column does not exist on the regular invoice: we must add it

![image.png](https://images.zenhubusercontent.com/595e669a61b4ca3b0a737135/02851205-0f02-4a28-aa64-676c0af1ab8d)

## Acceptance Criteria & Tests

_Scenario1_ for alternative invoice

1. Log in as enterprise
2. Check the unit price on the invoice is the same than in the shopfront
3. The unit price displayed should not change when the quantity ordered of the same product is changing

_Scenario 2_ for regular invoice

1. Log in as enterprise
2. See the unit price column is added on all invoices

**Comment by drummer83** [comment]
@RachL I don't agree to this one. At least in Germany the unit price is only required in the shopfront, not on the invoice. In my opinion it is correct to show the the price per quantity on the invoice.
I have never seen it differently.

**Comment by RachL** [comment]
@drummer83 hum this annoying. Because in FR the unit price is legally required. Unless we find a way to display both (which I'm unsure how to do without being confusing). I think we need to keep the legal one, even if only one country.

**Comment by RachL** [comment]
Maybe an idea but it would still be confusing during the transition: 
- we move unit price in place of current unit / final weight if BOM was used (current parenthesis)
- the unit / final weight if BOM was used is moved to a dedicated column
- we keep the price per quantity in a column

My example would give then: 

Melon TEST (0,11 € / sachet)  | quantity 1 | unit 1000 sachets | price per quantity 105 € | VAT 0 | Total 105 €

**Comment by drummer83** [comment]
Sound good to me, @RachL!
Thanks for your consideration. 🙏

**Comment by abdellani** [comment]
Hi @RachL 

I want to confirm something. Is the objective of this issue to add the unit prices next to the line item name?
![image](https://github.com/openfoodfoundation/openfoodnetwork/assets/16113997/9344d6b8-1142-4908-8645-d9685e167e78)


The following example is from the database seed:
![image](https://github.com/openfoodfoundation/openfoodnetwork/assets/16113997/b9248560-0be8-424d-80c4-09943baaf8d9)
![image](https://github.com/openfoodfoundation/openfoodnetwork/assets/16113997/49f22390-08ad-4067-a0d5-74217607b129)

Do we want to replace `1g` with (3300 $/kg ) in the invoice?

**Comment by abdellani** [comment]
Hi @RachL 
I made the changes that you asked for, but I think we need to show the unit used during the purchase (1g). 

![image](https://github.com/openfoodfoundation/openfoodnetwork/assets/16113997/34769584-bfba-4836-9fdf-80c8d7b50592)
![image](https://github.com/openfoodfoundation/openfoodnetwork/assets/16113997/77a2f0b9-855d-471a-8904-c39a7e2c1d8c)

**Comment by RachL** [comment]
@abdellani yes the "1g" is moved to its own column, see https://github.com/openfoodfoundation/openfoodnetwork/issues/11538
